teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>Is there a more generic way to connect elements with their atomic number? Right now this function would raise an error if the ligand contains a sulfur atom.</p><p> </p><p>Also, could we rename the atoms already before going to the openff molecule, so in the function before, in which we assign the bond orders?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>RDKit Molecules do not have a "Name" Attribute , but I discovered that they have a property with PDB Infos which includes the name</p>